src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -160,7 +160,7 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig, protected va
   /* Create a google bucket and populate it with init files */
   private[service] def initializeBucket(googleProject: GoogleProject, clusterName: ClusterName, bucketName: GcsBucketName, clusterRequest: ClusterRequest): Future[GcsBucketName] = {
     for {
-      _ <- gdDAO.createBucket(googleProject, bucketName)
+      _ <- gdDAO.createBucket(googleProject, bucketName, clusterRequest.serviceAccount)
```

Comment:
So this'll conflict with https://github.com/broadinstitute/leonardo/pull/58 because `serviceAccount` is no longer a property of `clusterRequest`. But it should be easy to reconcile.

Reply:
I can wait until yours is done, since the merge will be easy.